dev-python/python-nbxmpp/python-nbxmpp-9999.ebuild

```diff
@@ -0,0 +1,22 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+PYTHON_COMPAT=( python2_7 python3_{3,4,5} )
+
+inherit distutils-r1 mercurial
+
+TAG=8
+
+DESCRIPTION="Python library to use Jabber/XMPP networks in a non-blocking way"
+HOMEPAGE="http://python-nbxmpp.gajim.org/"
+EHG_REPO_URI="https://hg.gajim.org/python-nbxmpp"
+
+SLOT="0"
+LICENSE="BSD"
+KEYWORDS="**"
```

Comment:
KEYWORDS="**" is not correct here. KEYWORDS should be empty, or not present at all.